rover_msgs/Cameras.lcm

```diff
@@ -1,6 +1,5 @@
 package rover_msgs;
 
 struct Cameras {
-    int8_t port_0;
-    int8_t port_1;
+    int32_t port[4];
```

Comment:
can you add a comment for why 4 so when someone looks at this in a couple years they dont try to stream like 8 and get confused why it doesn't work

Reply:
yea sure